Kitodo-SRU-Import/pom.xml

```diff
@@ -40,6 +40,22 @@
             <version>2.0.6</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
Same here: Why you are using junit 4.12 instead of 5.4.1 and more not using the maven variable for this?